components/layouts/blog/blog-post.hbs

```diff
@@ -22,16 +22,10 @@
                 <time>10 minutes</time>
             </div>
             <div class="post-hero__time-social-links">
-                <a href="https://www.linkedin.com/company/aquent" title="LinkedIn" target="_blank"
-                    class="social-icon__link">
-                    <img class="social-icon--linkedin" /><span class="show-for-sr">LinkedIn</span>
-                </a>
-                <a href="https://twitter.com/Vitamin T" title="Twitter" target="_blank" class="social-icon__link">
-                    <img class="social-icon--twitter" /><span class="show-for-sr">Twitter</span>
-                </a>
-                <a href="https://www.facebook.com/aquent/" title="Facebook" target="_blank" class="social-icon__link">
-                    <img class="social-icon--facebook" /><span class="show-for-sr">Facebook</span>
-                </a>
+                 <a href="https://www.linkedin.com/company/aquent" title="LinkedIn" target="_blank"
+                    class="social-icon__link--linkedin"></a>
+                <a href="https://twitter.com/Vitamin T" title="Twitter" target="_blank" class="social-icon__link--twitter"></a>
+                <a href="https://www.facebook.com/aquent/" title="Facebook" target="_blank" class="social-icon__link--facebook"></a>
```

Comment:
These shouldn't be links to our profiles, they should be pre-formatted links for sharing a post. Look at the AQ AU blog as for the link formats.

Reply:
Updated link format.